docs/blog/2013-01-27-refinements101-reax.lhs

```diff
@@ -194,7 +194,7 @@ recursive calls --- we get the above by plugging the parameters
 \end{code}
 
 \begin{code} Finally, to check the output guarantee is met, LiquidHaskell asks the SMT solver to prove that
-(b >= 2n - 2)  =>  (b >= n)
+(b >= 2n - 3)  =>  (b >= n)
```

Comment:
This should be 3 instead of 2 right? In line 193 it was written with a 3 and also paragraph below isn't true anymore, if we write a 2 here:

> The SMT solver will refuse, of course, since the above implication is 
> *not valid* (e.g. when `n` is `2`) Thus, via SMT, LiquidHaskell proclaims
> that the function `fib'` does not implement the advertised type and hence
> marks it *unsafe*.

Reply:
Thanks for catching that!!!